python-backend/app/commands.py

```diff
@@ -1,25 +1,20 @@
-import random
 import click
-import names
 
-from concurrent.futures import ThreadPoolExecutor, as_completed
 from sqlalchemy.exc import DBAPIError
 from multiprocessing.dummy import Pool as ThreadPool
 
-from app import auth
 from app.api.utils.include.user_info import User
-from app.api.utils.random import generate_mine_no, generate_mine_name, random_geo, random_key_gen, random_date, random_region, random_mine_category
-from app.extensions import db, sched
-from app.scheduled_jobs import ETL_jobs
+from app.extensions import db
 
 from tests.factories import MineFactory, MinePartyAppointmentFactory
 
 
 def register_commands(app):
+
     @app.cli.command()
     def import_idir():
-        from app.scheduled_jobs.IDIR_jobs import _import_empr_idir_users
-        _import_empr_idir_users()
+        from app.scheduled_jobs.IDIR_jobs import import_empr_idir_users
```

Comment:
Did you preserve this only so it could be run via cli? Can it not be moved into scheduler?

Reply:
I see, we have removed all the schedulers and preserved the cli commands, and the central scheduler kicks up an instance of the pod and runs the cli command. maybe rename the module to be cli jobs?

Reply:
Yes! CLI seemed to be the cleanest way of running this and it also made sense as sort of a pseudo pattern.
-> Where we have a scheduled-jobs dir in each project(backend/nris) that contains the definition of the job, since it may need access to models and internal helper methods.
-> Then a cli invocation in commands file, which may have more logging and what not to invoke the functions in the scheduled jobs.
-> Then in scheduler we just spin up the container with a different entrypoint to run the job

Whereas moving the code to scheduler may end up introducing duplication with requirements.txt and code which i wasnt able to think of doing in a clean way

Reply:
@Jsyro renamed module to be cli_jobs, i agree it makes more sense than scheduled jobs